### ISSUE TITLE
Exception "hood closed" - NoneType object is not iterable

### ISSUE THREAD

**Issue by ChristophCaina** [lendy007/homeassistant-skodaconnect#151]
**Describe the bug**
I'm getting the following exception in the HA Logs:

```
Logger: homeassistant
Source: custom_components/skodaconnect/binary_sensor.py:52
Integration: Skoda Connect ([documentation](https://github.com/lendy007/homeassistant-skodaconnect/blob/main/README.md), [issues](https://github.com/lendy007/homeassistant-skodaconnect/issues))
First occurred: 14:29:23 (4 occurrences)
Last logged: 14:59:30
Error doing job: Task exception was never retrieved

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 168, in _handle_refresh_interval
    await self._async_refresh(log_failures=True, scheduled=True)
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 316, in _async_refresh
    self.async_update_listeners()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 121, in async_update_listeners
    update_callback()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 545, in async_write_ha_state
    self._async_write_ha_state()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 583, in _async_write_ha_state
    state = self._stringify_state(available)
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 551, in _stringify_state
    if (state := self.state) is None:
  File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 211, in state
    if (is_on := self.is_on) is None:
  File "/config/custom_components/skodaconnect/binary_sensor.py", line 52, in is_on
    return not self.instrument.is_on
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 188, in is_on
    return self.state
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 172, in state
    val = super().state
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 58, in state
    if hasattr(self.vehicle, self.attr):
  File "/usr/local/lib/python3.10/site-packages/skodaconnect/vehicle.py", line 2484, in hood_closed
    bonnet = next(item for item in doors if item['name'] == 'BONNET')
TypeError: 'NoneType' object is not iterable
```

**Latest working release**
unknown

**Debug logs**
If applicable, add debug logs. See [README](https://github.com/lendy007/homeassistant-skodaconnect#enable-debug-logging) on how to enable debug logs with response debugging.

**Installation:**
 - Home Assistant variant: [e.g. Hass OS, Docker, Core, Supervised]
 - Python version [e.g. 3.7, 3.8]
 - Python library versions ["python3 -m pip list"]

**Additional context**
do you need any additional information?

**Comment by Farfar** [comment]
> **Describe the bug** I'm getting the following exception in the HA Logs:
> 
> ```
> Logger: homeassistant
> Source: custom_components/skodaconnect/binary_sensor.py:52
> Integration: Skoda Connect ([documentation](https://github.com/lendy007/homeassistant-skodaconnect/blob/main/README.md), [issues](https://github.com/lendy007/homeassistant-skodaconnect/issues))
> First occurred: 14:29:23 (4 occurrences)
> Last logged: 14:59:30
> Error doing job: Task exception was never retrieved
> 
> Traceback (most recent call last):
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 168, in _handle_refresh_interval
>     await self._async_refresh(log_failures=True, scheduled=True)
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 316, in _async_refresh
>     self.async_update_listeners()
>   File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 121, in async_update_listeners
>     update_callback()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 545, in async_write_ha_state
>     self._async_write_ha_state()
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 583, in _async_write_ha_state
>     state = self._stringify_state(available)
>   File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 551, in _stringify_state
>     if (state := self.state) is None:
>   File "/usr/src/homeassistant/homeassistant/components/binary_sensor/__init__.py", line 211, in state
>     if (is_on := self.is_on) is None:
>   File "/config/custom_components/skodaconnect/binary_sensor.py", line 52, in is_on
>     return not self.instrument.is_on
>   File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 188, in is_on
>     return self.state
>   File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 172, in state
>     val = super().state
>   File "/usr/local/lib/python3.10/site-packages/skodaconnect/dashboard.py", line 58, in state
>     if hasattr(self.vehicle, self.attr):
>   File "/usr/local/lib/python3.10/site-packages/skodaconnect/vehicle.py", line 2484, in hood_closed
>     bonnet = next(item for item in doors if item['name'] == 'BONNET')
> TypeError: 'NoneType' object is not iterable
> ```
> 
> **Latest working release** unknown
> 
> **Debug logs** If applicable, add debug logs. See [README](https://github.com/lendy007/homeassistant-skodaconnect#enable-debug-logging) on how to enable debug logs with response debugging.
> 
> **Installation:**
> 
> * Home Assistant variant: [e.g. Hass OS, Docker, Core, Supervised]
> * Python version [e.g. 3.7, 3.8]
> * Python library versions ["python3 -m pip list"]
> 
> **Additional context** do you need any additional information?

Which library version? Should be fixed in 1.1.27.
From command line run "python3 -m pip show skodaconnect" or check startup logs for home assistant.

**Comment by Farfar** [comment]
#145 is related as well.
